Makefile

```diff
@@ -8,6 +8,7 @@ DIRS := dev etc home lib/modules media proc root \
 
 install:
 	$(foreach dir,$(DIRS),mkdir -p $(DESTDIR)/$(dir);)
+	ln -s /run $(DESTDIR)/var/run
```

Comment:
Could you make it relative?

Reply:
Now looking at it a bit more. base-files has an absolute symlink... Which mean core bases have that too. So maybe that is ok. Maybe I am the only one hating absolute symlinks pointing to something completely wrong when mounted in the wrong place. Though systemd does create it as relative.